clusterous/defaults.py

```diff
@@ -39,6 +39,9 @@
 taggable_name_max_length = 64       # Arbitrary but ample, keeping in mind AWS keys can be max 127 chars
 
 security_group_name_format = '{0}-sg'
+nat_name_format = '{0}-nat'
+nat_name_tag_value = 'nat'
+nat_instance_type = 't2.small'
```

Comment:
Any reason to make it a t2.small instead of a t2.micro? In our discussions we had said we could probably get away with a micro for now. Did you experience performance problems?